message_definitions/v1.0/storm32.xml

```diff
@@ -193,9 +193,6 @@ Documentation:
       <entry value="2048" name="MAV_STORM32_GIMBAL_DEVICE_FLAGS_RC_MIXED">
         <description>RC control. The RC input signal fed to the gimbal device is mixed into the gimbal's orientation. Is overriden by RC_EXCLUSIVE flag if that is also set.</description>
       </entry>
-      <entry value="65535" name="MAV_STORM32_GIMBAL_DEVICE_FLAGS_NONE">
```

Comment:
Upstream now pulls the ardupilot included files directly, so will get this and the following

Reply:
Cool